client/src/views/home/HomePage.vue

```diff
@@ -84,35 +86,47 @@ enum HomePageState {
 
 const notificationManager: NotificationManager = inject(NotificationKey)!;
 const storageManager: StorageManager = inject(StorageManagerKey)!;
+const hotkeyManager: HotkeyManager = inject(HotkeyManagerKey)!;
+const hotkeys = hotkeyManager.newHotkeys();
+
 const state = ref(HomePageState.OrganizationList);
 const storedDeviceDataDict = ref<{ [slug: string]: StoredDeviceData }>({});
 const selectedDevice: Ref<AvailableDevice | null> = ref(null);
-
 const routeWatchCancel = watchRoute(async () => {
   const query = getCurrentRouteQuery();
   if ('link' in query && query.link) {
     await openJoinByLinkModal(query.link as string);
   }
 });
+const currentModal: Ref<HTMLIonModalElement | null> = ref(null);
 
 onMounted(async () => {
   storedDeviceDataDict.value = await storageManager.retrieveDevicesData();
+  await hotkeys.add(',', Modifiers.Ctrl, Platforms.Desktop | Platforms.Web, openSettingsModal);
 });
 
-onUnmounted(() => {
+onUnmounted(async () => {
   routeWatchCancel();
 });
 
 async function openCreateOrganizationModal(): Promise<void> {
-  const modal = await modalController.create({
+  if (currentModal.value) {
```

Comment:
I don't think this is the right way of handling these cases. It adds way too much code with way too much states. I think a better way of doing it would be to disable/reenable the shortcuts. 

```typescript
async function openCreateOrganizationModal(): Promise<void> {
  shortcuts.disable();
  const modal = await modalController.create({
    component: CreateOrganizationModal,
    canDismiss: true,
    cssClass: 'create-organization-modal',
    backdropDismiss: false,
  });
  await modal.present();
  const { data, role } = await modal.onWillDismiss();
  await modal.dismiss();
  shotcuts.enable();
```
and just add some code in the HotkeyManager to check whether a shortcut group is enabled or not.

and if some shortcuts have to remain valid, we could always create multiple groups and only disable one of them:
```typescript
const modalShortcuts = hotkeyManager.newHotkeys();
const otherShortcuts = hotkeyManager.newHotkeys();

modalShortcuts.add(blabla, openSettings);
modalShortcuts.add(blabla, openJoinOrg);
otherShortcuts.add(blabla, doSomething);

modalShortcuts.disable();
// open modal
modalShortcuts.enable();
```